File: ChibuzorTasks/RaiseToPower.java
import java.util.Scanner;

public class RaiseToPower{
	public static void main( String[] args){
		Scanner input = new Scanner(System.in);
		System.out.print("Enter the first Number: ");
		int firstNumber = input.nextInt();
		System.out.print("Enter the exponent for the first number: ");
		int exponent = input.nextInt();
		
		double result = 1;
		for (int i = 0; i < exponent; i++) {
			result *= firstNumber;
		}
		
		System.out.printf("%d raised to the power of %d is %.2f", firstNumber, exponent,result);

	
	}



}

File: ChibuzorTasks/raise_to_power.py
first_number = int(input("Enter the first number: "))
exponent = int(input("Enter an exponent to raise the first number by: "))

result = first_number
for i in range(1,exponent) :
	result *= first_number

print(f"{first_number} raised to the power of {exponent} is {result}")